### DISCUSSION TITLE
Error: Cannot read properties of undefined (reading 'isRTL')

### DISCUSSION THREAD

**Discussion by frimen** [requarks/wiki#5280] [Error / Bug Report]
Hello everyone.

When trying to install another language, it gives the following error: Error: Cannot read properties of undefined (reading 'isRTL'). 

![Скриншот 11-05-2022 174856](https://user-images.githubusercontent.com/4110079/167832717-499afe99-854b-444b-94eb-9e63736a0bfb.png)

System: Ubuntu 20.04 server

How to overcome this problem.

**Comment by nyafi24** [comment]
Hi, check Wikijs logs. Most likely an error when inserting into the database (postgres) due to invalid encoding. 

In this case, you can try to recreate the template1 (which used for create new databases) with the necessary encoding:
```
# Before that don't forget to make a backup just in case.
UPDATE pg_database SET datistemplate=false WHERE datname = 'template1';
DROP DATABASE template1;
CREATE DATABASE template1 WITH owner=postgres ENCODING='UTF-8' lc_collate='en_US.utf8' lc_ctype='en_US.utf8' template template0;
COMMENT on database template1 IS 'default template for new databases';
UPDATE pg_database SET datistemplate=true WHERE datname='template1';
```
[Source](https://evileg.com/ru/post/2/)

After that, you need to recreate the database for wikiJS.
